File: 2740-find-the-value-of-the-partition/2740-find-the-value-of-the-partition.js
/**
 * @param {number[]} nums
 * @return {number}
 */

function findValueOfPartition(nums){
    nums.sort((a,b)=>a-b);
    //find min difference among them
    let ans = Number.MAX_SAFE_INTEGER;
    for(let i=1;i<nums.length;i++){
        ans=Math.min(ans,nums[i]-nums[i-1]);
    }
    return ans;
}

File: 2740-find-the-value-of-the-partition/2740-find-the-value-of-the-partition.cpp
class Solution {
public:
    int findValueOfPartition(vector<int>& nums) {
        int n=nums.size();
        sort(begin(nums),end(nums));
        int ans=INT_MAX;
        for(int i=1;i<n;i++){
            ans=min(ans,nums[i]-nums[i-1]);
        }
        return ans;
    }
};